arbitrum-docs/intro/intro.md

```diff
@@ -4,10 +4,10 @@ description: 'Arbitrum is a suite of Ethereum Layer-2 scaling solutions that you
 author: dzgoldman
 ---
 
-import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md';
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.mdx';
 
 <PublicPreviewBannerPartial />
 
-import GentleIntroPartial from '../partials/_gentle-intro-partial.md';
+import GentleIntroPartial from '../partials/\_gentle-intro-partial.md';
```

Comment:
I don't think we need that `\` there, can you verify?

Reply:
nice catch! Re-tweaking my `prettierrc` to address this issue.